### PR TITLE
8339845: Update color.org and wapforum.org links to use HTTPS instead of HTTP

### PR DESCRIPTION
**Pull request by nizarbenalla**
Can I get a review for this doc-only change? I updated some links because the websites moved from HTTP to HTTPS. The redirects can cause some false-positives on tests.

TIA

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8339845](https://bugs.openjdk.org/browse/JDK-8339845): Update color.org and wapforum.org links to use HTTPS instead of HTTP (**Sub-task** - P4)

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)
 * [Harshitha Onkar](https://openjdk.org/census#honkar) (@honkar-jdk - **Reviewer**)
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21001/head:pull/21001` \
`$ git checkout pull/21001`

Update a local copy of the PR: \
`$ git checkout pull/21001` \
`$ git pull https://git.openjdk.org/jdk.git pull/21001/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21001`

View PR using the GUI difftool: \
`$ git pr show -t 21001`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21001.diff">https://git.openjdk.org/jdk/pull/21001.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21001#issuecomment-2349181773)

### CODE REVIEW

**Review by prrace — APPROVED** [review]

**Review by abdelhak-zaaim — COMMENTED** [review]
The update to HTTPS is appreciated for consistency, though not strictly necessary since HTTP links already redirect to HTTPS. Please ensure all updated links function as expected and align with the project standards.

File: src/java.desktop/share/classes/java/awt/image/ColorConvertOp.java [hunk 1]
```diff
@@ -690,7 +690,7 @@ private int getRenderingIntent (ICC_Profile profile) {
          * shall be set to zero. Thus, we are ignoring two most significant
          * bytes here.
          *
-         *  See http://www.color.org/ICC1v42_2006-05.pdf, section 7.2.15.
+         *  See https://www.color.org/ICC1v42_2006-05.pdf, section 7.2.15.
```
Comment (honkar-jdk) [inline src/java.desktop/share/classes/java/awt/image/ColorConvertOp.java:693]: @prrace Current ICC doc links point to older ICC Specification document. Do they need to be updated to the latest one?
https://www.color.org/specification/ICC.1-2022-05.pdf

File: src/java.desktop/share/classes/javax/imageio/plugins/tiff/BaselineTIFFTagSet.java [hunk 2]
```diff
@@ -1008,7 +1008,7 @@ public final class BaselineTIFFTagSet extends TIFFTagSet {
     /**
      * Constant specifying the "ICC Profile" tag.
      *
-     * @see <a href="http://www.color.org/ICC1V42.pdf">ICC Specification, section B.4: Embedding ICC profiles in TIFF files</a>
+     * @see <a href="https://www.color.org/ICC1V42.pdf">ICC Specification, section B.4: Embedding ICC profiles in TIFF files</a>
```
Comment (honkar-jdk) [inline src/java.desktop/share/classes/javax/imageio/plugins/tiff/BaselineTIFFTagSet.java:1011]: Same here ?

File: src/java.desktop/share/classes/java/awt/image/ColorConvertOp.java (hunk 1 above)
Comment (prrace) [reply src/java.desktop/share/classes/java/awt/image/ColorConvertOp.java:693]: That would be out of scope for this.

**Review by honkar-jdk — APPROVED** [review]

**Review by aivanov-jdk — APPROVED** [review]

**Comment by aivanov-jdk** [comment]
@nizarbenalla Looks you're a committer now. You have to issue the `/integrate` command again.

**Comment by nizarbenalla** [comment]
I assumed this would be automatically merged once I became a committer.

Thank you all for the reviews!

/integrate

**Comment by aivanov-jdk** [comment]
> I assumed this would be automatically merged once I became a committer.

No, the Skara bots don't revisit the PRs after the OpenJDK Census is changed.

You weren't a committer when you issued the `/integrate` command, so it required a sponsor. By the time I decided to sponsor, you're already a committer, and no one can sponsor integrating any more.

**Pull request closed without merging**